meta/saiserialize.cpp

```diff
@@ -1934,6 +1934,13 @@ void sai_deserialize_enum(
         }
     }
 
+    // for backward compatibility from SAI v1.6
+    if (s == "SAI_NEXT_HOP_GROUP_TYPE_ECMP")
```

Comment:
i can see see this in the SAI header, why do we need this?

Reply:
This does in fact squash a test failure that I was observing. Not an argument for or against, just observation.

Reply:
this is fix for sai header 1.6 since that enum was marked as ignore, and is there just for backward compatibility to compile, but is no longer added to metadata to check for deserialize